cleaned_data/modsbyPID/rftaart:10.xml

```diff
@@ -10,12 +10,7 @@
       <title>Sacrificing meaningful items during evacuation</title>
    </titleInfo>
    <originInfo>
-      <dateIssued>2021</dateIssued>
-      <dateIssued encoding="edtf" keyDate="yes">2021</dateIssued>
```

Comment:
@logantillman - We want to keep the date value with the edtf encoding, but we simply need to change the date element to dateIssued. Can you run this through one more time and change the export template to include this date but with the correct element?